nix/shared.nix

```diff
@@ -563,6 +563,8 @@ in
   rec {
     inherit trivial;
 
+    nixpkgs = pkgs;
```

Comment:
I suggest `inherit pkgs;`, mainly because I try to reserve `nixpkgs` to refer to the Nixpkgs source code, rather than the package set